projects/opensips/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Google LLC
+# Copyright 2022 Google LLC
```

Comment:
nit: please leave copyright year 2021

Reply:
Sure!  Our of curiosity, is it because A) 2021 is correct, or B) a single change per PR is better?

Reply:
A) 2021 is correct